builtin/providers/aws/resource_aws_network_interface.go

```diff
@@ -46,6 +46,12 @@ func resourceAwsNetworkInterface() *schema.Resource {
 				Set:      schema.HashString,
 			},
 
+			"source_dest_check": &schema.Schema{
+				Type:     schema.TypeBool,
+				Optional: true,
+				Computed: true,
```

Comment:
which also means that we need to remove the `Computed` part here (can't have both computed and default)